net/v2ray-geodata/Makefile

```diff
@@ -51,23 +51,23 @@ define Package/v2ray-geoip
   $(call Package/v2ray-geodata/template)
   TITLE:=GeoIP List for V2Ray
   PROVIDES:=v2ray-geodata xray-geodata xray-geoip
-  VERSION:=$(GEOIP_VER)-r$(PKG_RELEASE)
+  SOURCE_VERSION:=$(GEOIP_VER)-r$(PKG_RELEASE)
```

Comment:
The aim here is to set the package version (PKG_VERSION), not for git submodules.

Reply:
but is it actually changed?

Reply:
ok will revert!